apps/pie-microsite/src/assets/styles/plugins/_prism.scss

```diff
@@ -121,7 +121,7 @@ div.code-toolbar {
   position: relative;
   max-width: 1025px;
   // overflow: hidden;
-  margin: 25px 0px 40px;
+  margin: 24px 0px;
```

Comment:
can you use `f.spacing(e)` here please?

Reply:
for reference: https://github.com/justeattakeaway/fozzie-components/blob/7fd01766b3126b30b1ab704c131b8ac767f882ea/packages/tools/fozzie/src/scss/settings/_variables.scss#L94